build.sbt

```diff
@@ -20,5 +20,12 @@ libraryDependencies += "com.mohiva" %% "play-silhouette-persistence" % "5.0.7"
 libraryDependencies += "net.codingwell" %% "scala-guice" % "4.1.0"
 libraryDependencies += "software.amazon.awssdk" % "aws-sdk-java" % "2.7.11"
 
-libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.2" % Test
+//libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.2" % Test
 libraryDependencies +=  "com.github.jarlakxen" %% "drunk" % "2.5.0"
+libraryDependencies ++= Seq(
+  "com.mohiva" %% "play-silhouette-testkit" % "5.0.7" % "test"
+)
+libraryDependencies ++= Seq(
+  "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.2" % "test"
```

Comment:
Why does this now need to be wrapped in `Seq`?

Also, does `% Test` do the same thing as `% "test"`? I'm not familiar enough with sbt to know.

Reply:
I've no idea, it was like this in the documentation and changed it out of desperation at one point. It works though so I've deleted the commented line.